### PR TITLE
Std args json

### PR DESCRIPTION
**Pull request by InnocentBug**
# Description

This rehandles the std fields in JSON.

1) it add default fields when they are not sent by the API. This helps to tolerate missing required arguments.
2) default fields are better suppressed when serializing, this helps to reduce false positives during integration testing.

## Notes
This required to change some default list fields from `None` to list.
## Checklist

- [x] My name is on the list of contributors (`CONTRIBUTORS.md`) in the pull request source branch.
- [x] I have updated the documentation to reflect my changes.

### CODE REVIEW

File: src/cript/nodes/primary_nodes/collection.py [hunk 1]
```diff
@@ -59,10 +59,10 @@ class JsonAttributes(PrimaryBaseNode.JsonAttributes):
         # TODO add proper typing in future, using Any for now to avoid circular import error
         member: List[User] = field(default_factory=list)
         admin: List[User] = field(default_factory=list)
-        experiment: Optional[List[Any]] = None
-        inventory: Optional[List[Any]] = None
+        experiment: List[Any] = field(default_factory=list)
```
Comment (nh916) [inline src/cript/nodes/primary_nodes/collection.py:62]: oh I see, so by default you gave it a list in case not provided by API

**Comment by nh916** [comment]
3 thoughts

1. is this safe to add before ACS?
2. are there any known errors that could come up?
3. where are you getting the default fields for the node?

File: src/cript/nodes/primary_nodes/collection.py (hunk 1 above)
Comment (InnocentBug) [reply src/cript/nodes/primary_nodes/collection.py:62]: Yes, exactly

**Comment by InnocentBug** [comment]
> 3 thoughts
> 
> 1. is this safe to add before ACS?
> 2. are there any known errors that could come up?
> 3. where are you getting the default fields for the node?

1) yes. Once we fixed the bugs in the tests. It actually will help, because it makes pulling nodes from api more easy. Less error for users
2). It that I know of. The failing tests is just that we am have to adjust the test data we compare to
3) we get the default fields from the default values of the data classes. At some point it would be ideal to set those with the help of the db schema. But for now they are hard coded.

**Comment by nh916** [comment]
tested this locally against staging, and it was good to go! ✅

**Review by nh916 — APPROVED** [review]

**Review by nh916 — APPROVED** [review]

**Pull request merged**